ktlint/src/test/resources/TestBaselineFile._kt

```diff
@@ -0,0 +1,5 @@
+// This class is stored on purpose in a file having extension "_kt". This prevents the file from being autocorrected by
```

Comment:
I think we can just exclude this file (and also `TestBaselineExtraErrorFile`) via negation from the task execution (like `!ktlint/src/test/resources/TestBaselineFile.kt`)